db/migrate/20181116104823_fixed_ignore_taxonomies_for_audit_filters.rb

```diff
@@ -2,7 +2,7 @@ class FixedIgnoreTaxonomiesForAuditFilters < ActiveRecord::Migration[5.2]
   def up
     if (permission = Permission.find_by_name('view_audit_logs'))
       permission.filters.uniq.each do |filter|
-        next if filter.role.locked?
+        next if filter&.role&.locked?
```

Comment:
Looks like now it is failing in the `save!` which triggers a validation assuming there is a role. 
I guess we need to do 
```suggestion
        next if filter.role.nil? || filter.role.locked?
```